dependencies/bionic/gitlab-sidekiq-fetcher/copyright

```diff
@@ -0,0 +1,26 @@
+Format: http://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: gitlab-sidekiq-fetcher
+Upstream-Contact: Valery Sizov <valery@gitlab.com>
+Source: https://gitlab.com/gitlab-org/sidekiq-reliable-fetch
+
+Files: *
+Copyright: 2016, Rémi Bauzac
+License: LGPL-3.0
```

Comment:
Is this all correct? I wasn't sure about who is the copyright holder so I picked the one who put the license in the repo.